recipes/qtmockwebserver/all/conandata.yml

```diff
@@ -0,0 +1,12 @@
+sources:
+  cci.20211015:
```

Comment:
Note that the new version policy for cases like this is to have
`0.0.0.cci.20211015` so version ranges behave correctly if a version is ever released, thanks!

Reply:
fixed. thanks you